sympy/solvers/bivariate.py

```diff
@@ -149,17 +149,19 @@ def _lambert(eq, x):
     sol = []
     # check only real solutions:
     for k in [-1, 0]:
```

Comment:
I don't get what you are trying to get me to do here?

Reply:
Instead of using `k` (which is the name of the 2nd argument for `LambertW` -- but someone won't know that unless they look at the docstring of `LambertW`) you could use `for branch in [-1, 0]:`.